eng/pipelines/templates/jobs/archetype-sdk-mgmt.yml

```diff
@@ -64,3 +75,8 @@ jobs:
           testRunTitle: "$(OSName) Test Results"
           testResultsFormat: "VSTest"
           mergeTestResults: true
+      - task: PublishBuildArtifacts@1
```

Comment:
is it possible to also archive .trx files? Seems those are not getting archived, because those files have results of which tests passed and which one failed.

Reply:
@shahabhijeet , the task PublishTestResults already takes care of your concern, the detail run tests result was published to Runs under Test Plans in Azure DevOps.